clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/RedisTaskRepository.java

```diff
@@ -154,11 +156,19 @@ public Task getByClientRequestId(String clientRequestId) {
     }), "Getting all running tasks");
   }
 
+  @Override
+  public List<Task> listByThisInstance() {
+    return list().stream()
+        .filter(t -> t.getOwnerId().equals(ClouddriverHostname.ID))
```

Comment:
Can `getOwnerId()` return `null` if the task was created before this change lands? If so, can probably just swap the order of the operands.

Reply:
Good point